launch_testing/launch_testing_examples/launch_testing_examples/set_param_launch_test.py

```diff
@@ -43,26 +43,25 @@ def generate_test_description():
 if os.name != 'nt':
     class TestFixture(unittest.TestCase):
 
-        def test_set_parameter(self, proc_output):
+        def setUp(self):
             rclpy.init()
-            node = Node('test_node')
-            response = set_parameter(node, value=True)
-            assert response.successful, 'Could not set parameter!'
-            rclpy.shutdown()
+            self.node = Node('test_node')
 
+        def tearDown(self):
+            rclpy.shutdown()
```

Comment:
do we need `destroy_node` here?

```suggestion
            self.node.destroy_node()
            rclpy.shutdown()
```

Reply:
Yes! 100%. That was part of what I was trying to fix here, I can't believe I forgot it. Thanks for pointing it out.